tests/test_unicodecsv.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+
+import unicodecsv
+
+class TestUnicodeCsv(unittest.TestCase):
+
+    def test_unicode(self):
```

Comment:
I am not sure we care for a test of unicodecsv. What we need is a test of the support of unicode in thsi spdx library instead. For inatnce, what if we pass unicode content? etc.